gulp/modules/CssComb.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
```

Comment:
For the sake of maintainability, please add a comment to the top of this file briefly explaining the reason for including an inline fork of CSSComb. I know we discussed in person, but if others in the future need to revisit this, it'll be helpful for them.